packages/Python/lldbsuite/test/lang/swift/implementation_only_imports/library_indirect/TestLibraryIndirect.py

```diff
@@ -0,0 +1,70 @@
+# TestLibraryIndirect.py
+#
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2019 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See https://swift.org/LICENSE.txt for license information
+# See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+#
+# ------------------------------------------------------------------------------
+"""
+Test `@_implementationOnly import` behind some indirection in a library used by the main executable
+"""
+import commands
+import lldb
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+import lldbsuite.test.lldbutil as lldbutil
+import os
+import os.path
+import time
+import unittest2
+
+class TestLibraryIndirect(TestBase):
+
+    mydir = TestBase.compute_mydir(__file__)
+    
+    @swiftTest
```

Comment:
@JDevlieghere do we still use this decorator for anything?

Reply:
Yep, it ensures we skip everything but macOS/Linux and no i386.